source/environments/Pict-Environment-Object.js

```diff
@@ -203,6 +218,41 @@ class PictEnvironmentObject
 
 		return true;
 	}
+
+	customReadClassFunction (pAddress, pClass)
+	{
+		let tmpContent = this.contentMap._CLASS_LIST.indexOf(pClass) > -1 ? true : false;
+
+		if (this.storeEventLog) this.eventLog.Assign.push(this.createEventLogEntry(pAddress, pClass));
+		this.pict.log.info(`Mocking an CLASS READ for Address -> [${pAddress}]::[${pClass}]`, {Content: tmpContent});
+		return tmpContent;
+	}
+
+	customSetClassFunction (pAddress, pClass)
+	{
+		let tmpContent = this.contentMap._CLASS_LIST;
+
+		if (!this.contentMap._CLASS_LIST.indexOf(pClass) > -1)
+		{
+			this.contentMap._CLASS_LIST.push(pClass);
+		}
+
+		if (this.storeEventLog) this.eventLog.Assign.push(this.createEventLogEntry(pAddress, pClass));
+		this.pict.log.info(`Mocking an CLASS SET for Address -> [${pAddress}]::[${pClass}]`, {Content: tmpContent.join()});
```

Comment:
nit: 
```suggestion
		this.pict.log.info(`Mocking an CLASS SET for Address -> [${pAddress}]::[${pClass}]`, {Content: tmpContent.join(' ')});
```

We should join a list of classes with a space or some delimiter. I chose space because it matches how web does it.